spread.yaml

```diff
@@ -556,7 +556,7 @@ prepare: |
     fi
     if [[ "$SPREAD_SYSTEM" == opensuse-* ]]; then
         # refresh metadatadata
-        zypper ref
+        zypper --no-gpg-checks ref
```

Comment:
This should be  `--gpg-auto-import-keys` which will do the right thing. We still want the signatures to be checked.

Reply:
done